docs/modules/release-notes/pages/all-releases/4_13_2S.adoc

```diff
@@ -0,0 +1,115 @@
+= Kobiton 4.13.2S release notes
+:navtitle: Kobiton 4.13.2S release notes
+

```

Comment:
Please add the release date like other release notes. For this release, the date is June 20, 2024

Reply:
Changes made.